packages/server/migrations/20230914192911_add-archive-date-to-grants.js

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
```
bash-5.1# yarn db:migrate
yarn run v1.22.17
$ knex migrate:latest
Using environment: development
Batch 4 run: 1 migrations
Done in 12.76s.
bash-5.1# yarn db:rollback
yarn run v1.22.17
$ knex migrate:rollback
Using environment: development
Batch 4 rolled back: 1 migrations
Done in 13.01s.
bash-5.1# 
```

Reply:
## Before
<img width="600" alt="image" src="https://github.com/usdigitalresponse/usdr-gost/assets/6497366/762fa362-8686-430c-b544-ed922e1315e6">


## After
<img width="437" alt="image" src="https://github.com/usdigitalresponse/usdr-gost/assets/6497366/962b8ba8-7dc8-4239-8ab9-9b897c7369ef">

Reply:
Added separately in #migra